app/src/main/java/org/schabi/newpipe/settings/ContentSettingsManager.kt

```diff
@@ -91,6 +91,10 @@ class ContentSettingsManager(private val fileLocator: NewPipeFileLocator) {
                         is String -> {
                             preferenceEditor.putString(key, value)
                         }
+                        is HashSet<*> -> {
```

Comment:
```suggestion
                        is Set<*> -> {
```
Set exactly matches the type of ``putStringSet`` - so let's use it.